src/System.Management.Automation/engine/Modules/ModuleUtils.cs

```diff
@@ -16,35 +16,31 @@ internal static class ModuleUtils
         //  - Ignore files/directories when access is denied;
         //  - Search top directory only.
         private static readonly System.IO.EnumerationOptions s_defaultEnumerationOptions =
-                                        new System.IO.EnumerationOptions() { AttributesToSkip = 0 };
+                                        new System.IO.EnumerationOptions() { AttributesToSkip = System.IO.FileAttributes.Hidden };
```

Comment:
We already have `using System.IO;`
```suggestion
                                        new System.IO.EnumerationOptions() { AttributesToSkip = FileAttributes.Hidden };
```